R/rlistings.R

```diff
@@ -418,9 +425,15 @@ setMethod(
       bodymat
     )
 
+    col_alignment_values <- sapply(listing, obj_align)
+    colnames_align <- if (isFALSE(align_coltitles_as_cols(obj))) {
+      rep("center", length(cols))
+    } else {
+      col_alignment_values
+    }
     colaligns <- rbind(
-      rep("center", length(cols)),
-      matrix(sapply(listing, obj_align),
+      colnames_align,
+      matrix(col_alignment_values,
```

Comment:
so this will allow us to post process a listing, and reset the alignment right?

Reply:
Yes it does!